app/models/concerns/orchestration/common.rb

```diff
@@ -5,4 +5,9 @@ def handle_validation_errors
     logger.debug "Error occured during validations of #{self.class.name}: #{e.message}"
     nil
   end
+
+  # override sql logger for all orchestration mix-ins
+  def logger
```

Comment:
Yeah you are right, AR uses the new Rails 4 thing (notifications). I think I should be able to iterate through all our model classes and override the method. Any better ideas?